Add magdeburger developer days (#2005)
mainmatter/mainmatter.com: src/calendar/2023-05-08-magdeburger-developer-days.md

---
title: "Magdeburger Developer Days"
image: "/assets/images/calendar/2023-05-08-magdeburger-developer-days/mdd-logo-background.png"
location: "Magdeburg"
url: https://md-devdays.de/home
kind: conference
color: aqua
---

> Magdeburger Developer Days is a 3-day developer & community conference in

Reviewer: ```suggestion
Magdeburger Developer Days is a 3-day software engineering conference in
```

Magdeburg. Mainmatter Founder Marco Otte-Witte will talk about Continuous
Deployment Workflows. He will present main advantages compared to classical
processes, explain the essential components of such a process (especially with

> ryegard to testing and automation), and discuss typical challenges and

Author: ```suggestion
regard to testing and automation), and discuss typical challenges and
```

> Magdeburg. This year, Mainmatter Founder Marco Otte-Witte will talk about
> Continuous Deployment Workflows. In his talk, he will present main advantages
> compared to classical processes, explain the essential components of such a
> process (especially with ryegard to testing and automation), and discuss typical
> challenges and strategies to overcome them.

Reviewer: ```suggestion
Magdeburg. Mainmatter Founder Marco Otte-Witte will talk about
Continuous Deployment Workflows. He will present main advantages
compared to classical processes, explain the essential components of such a
process (especially with ryegard to testing and automation), and discuss typical
challenges and strategies to overcome them.
```
